job_bundles/Using_Files_In_Jobs.md

```diff
@@ -0,0 +1,1145 @@
+# Using Files in Your Jobs
+
+Many of the jobs that you submit to AWS Deadline Cloud will have input and output in the form
+of files. Your input files and output directories may be located on a combination of your shared filesystems
+and local drive. Your jobs require a way to locate the content in those locations. Deadline Cloud's
+[job attachments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html) and
+[storage profiles](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-shared.html) features work
+in concert to help your jobs locate the files that they needs.
+
+Deadline Cloud's job attachments feature helps you move files to your worker hosts from filesystem locations on your
+workstation that are not available on your worker hosts, and vice versa. It shuttles files between hosts using
+[Amazon S3](https://aws.amazon.com/pm/serv-s3/) as an intermediary. Job attachments can be enabled individually
+on each of your queues to make it available to jobs in those queues.
+
+You use Deadline Cloud's storage profiles to model the layout of shared filesystem locations on your workstation and
+worker hosts. This helps your jobs locate shared files and directories when their locations differ between your workstation
+and worker hosts, such as in cross-platform setups with Windows based workstations and Linux based worker hosts.
+Storage profile's model of your filesystem configuration is also leveraged by job attachments to identify which files
+it needs to shuttle between hosts through Amazon S3.
```

Comment:
One small suggestion, should we add something like "Service Managed Fleets work exclusively via Job Attachments accessed via S3. Customer Managed Fleets can leverage Job Attachments and Storage profiles together for local and network attached storage". This may not be clear for a new reader / user to deadline.

This makes it

Reply:
Good suggestion. Thanks